### ISSUE TITLE
Misspelled method name: GoogleMapView.addMapInializedListener()

### ISSUE THREAD

**Issue by looqmaan** [rterp/GMapsFX#141]
Can we please correct this misspelled method name? I believe it should be called: **addMapInitializedListener**.

```java

    public void addMapInializedListener(MapComponentInitializedListener listener) {
        synchronized (mapInitializedListeners) {
            mapInitializedListeners.add(listener);
        }
    }

```

**Comment by GeoffCapper** [comment]
Thanks @looqmaan it certainly should be. @rterp are we still unconcerned by breaking compatibility?

**Comment by rterp** [comment]
Yea we can go ahead and change it.

**Comment by GeoffCapper** [comment]
Fixed with commit #142

**Comment by looqmaan** [comment]
Thank you for the quick fix, guys! Appreciate that.
